jetson-utilsmake中にfatal error: GL/glew.h: No such file or directory

# Describe the bug
<!-- A clear and concise description of what the bug is. -->
jetson-utilsのmake中にエラーが発生しました。
```
fatal error: GL/glew.h: No such file or directory
```

# Environments
- OS: Ubuntu 18.04
- Software Version: https://github.com/dusty-nv/jetson-utils/tree/ebab1914877a51d4d33fa9b1f01b168adb712a32

# To Reproduce
<!-- Steps to reproduce the behavior -->

```
git clone https://github.com/dusty-nv/jetson-utils.git
mkdir jetson-utils/build
cd jetson-utils/build
cmake ..
make
```

# Expected behavior
<!-- A clear and concise description of what you expected to happen. -->
errorなくmakeが終了する

# Full log or Screenshots
<!-- If applicable, add screenshots to help explain your problem. -->
```
In file included from /home/jetbot/repos/jetson-utils/cuda/cudaPointCloud.cpp:26:0:
/home/jetbot/repos/jetson-utils/build/aarch64/include/jetson-utils/glUtility.h:27:10: fatal error: GL/glew.h: No such file or directory
 #include <GL/glew.h>
          ^~~~~~~~~~~
compilation terminated.
CMakeFiles/jetson-utils.dir/build.make:4188: recipe for target 'CMakeFiles/jetson-utils.dir/cuda/cudaPointCloud.cpp.o' failed
make[2]: *** [CMakeFiles/jetson-utils.dir/cuda/cudaPointCloud.cpp.o] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/jetson-utils.dir/all' failed
make[1]: *** [CMakeFiles/jetson-utils.dir/all] Error 2
Makefile:129: recipe for target 'all' failed
make: *** [all] Error 2
```

# Additional context
<!-- Add any other context about the problem here. -->

--- a/README.md
+++ b/README.md
@@ -17,12 +17,20 @@ This ROS package is designed to acquire images from the one or two CSI cameras a
 
 ### 1. Download and install [dusty-nv/jetson-utils](https://github.com/dusty-nv/jetson-utils)
 
+Check if it is already installed.
+
+```
+ldconfig -p | grep jetson && echo "installed" || echo "not installed yet"
+```
+
+If it is not installed, follow the steps below to install it.
+
 ```
 sudo apt install libglew-dev
 git clone https://github.com/dusty-nv/jetson-utils.git
 mkdir jetson-utils/build
 cd jetson-utils/build
-cmake ..
+cmake -DENABLE_NVMM=OFF ..
 make
 sudo make install
 sudo ldconfig

